fhir2dataset/api_caller.py

```diff
@@ -15,6 +15,8 @@
 
 def concat_cell(dataset, cols_name, element):
     # column of one row, this single cell is of the same type as element.value, i.e. a list
+    if element.value == []:
+        element.value = [None]
```

Comment:
why do you have to do that ? seems weird

Reply:
every column is in a list and each element too, which explains the [[None]]